src/scss/login.scss

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;600;700&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+.login {
+    color: black;
+}
+form {
+    padding: 30px;
+    max-width: 30em;
+    margin: 0 auto;
+}
+
+input[type=email], input[type=password] {
+    width: 100%;
+    height: 3em;
+    border-radius: 8px;
+    border: 1px solid #f5f5f5;
+    background-color: #f5f5f5;
+    box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.06), 0px 1px 3px rgba(0, 0, 0, 0.1)
+}
+input[type=checkbox], label {
+    float: left;
+}
+
+input[type=checkbox] {
+    margin-top: 1em;
+    width: 1.4em;
+    height: 1.4em;
+}
+input[type=password]{
+    background-image: url(/src/assets/crossed-out-eye.svg);
+    background-position: 380px 10px;
+    background-repeat: no-repeat;
+    background-size: 20px 20px;
+}
+
+.submit {
```

Comment:
A better name could be used here, say .submit--login 

This way, you can style all "submit" elements inside .submit and those specific to login with .submit--login

Reply:
change .submit to .submit--login